resources/views/checkout/partials/sidebar/summary.blade.php

```diff
@@ -1,7 +1,7 @@
 <x-rapidez-ct::card class="relative">
-    <div class="absolute inset-x-0 bottom-full -translate-y-6 max-md:hidden [&>*]:h-auto [&>*]:max-h-20 [&>*]:w-full [&>*]:object-contain">
+    <a href="/" class="absolute inset-x-0 bottom-full -translate-y-6 max-md:hidden [&>*]:h-auto [&>*]:max-h-20 [&>*]:w-full [&>*]:object-contain">
```

Comment:
This will cause issues with sites containing a subfolder.
To solve this it's better to use
`href="{{ url('/') }}"`
like done here https://github.com/rapidez/core/commit/ffada5151034709ff58ab68eeb2819288e8457e4#diff-46b75c2e4f87eb7c5fb5f7b41312df77f71a81f6a93a00b5807de74177774318R2